manim/camera/three_d_camera.py

```diff
@@ -289,7 +289,6 @@ def project_points(self, points):
                 factor = distance / (distance - zs)
                 factor[(distance - zs) < 0] = 10 ** 6
             points[:, i] *= factor
-        points = points + frame_center
```

Comment:
**(question)**: Did this line cause any bugs to occur or was it just redundant?

Reply:
This line caused the bug of the frame seemingly being centered at the origin regardless of the change of frame_center.